src/xcode/ENA/ENA/Source/HTTPClient/Resources/RegistrationToken/RegistrationReceiveModel.swift

```diff
@@ -2,7 +2,7 @@
 // 🦠 Corona-Warn-App
 //
 
-struct SubmissionTANModel: Codable {
+struct RegistrationReceiveModel: Codable {
```

Comment:
Why is it called 'Registration' when its 'submissionTAN' inside?

Reply:
Good catch! There was a renaming failure. registrationToken and teleTan got swapped. I corrected this.